bot/exts/moderation/infraction/management.py

```diff
@@ -420,6 +421,13 @@ def infraction_to_string(self, infraction: t.Dict[str, t.Any]) -> str:
         else:
             dm_sent_text = "Yes" if dm_sent else "No"
 
+        if jump_url == "":
+            # Infraction was issued prior to jump urls being stored in the database.
+            jump_url = "N/A"
+        elif "discord.com" in jump_url:
+            jump_url = f"[Click here.]({jump_url})"
+            # Else, infraction was issued in ModMail category.
```

Comment:
This would be simplified if we went with the "it's either a link or null"

Reply:
Yeah, I did think of that.